CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 2023-XX-XX
 
+### All SDKs
+* [FIXED][6602](https://github.com/stripe/stripe-android/pull/6602) Fixed an issue which caused a compiler error (duplicate class) when including payments *and* identity SDKs.
```

Comment:
```suggestion
* [FIXED][6602](https://github.com/stripe/stripe-android/pull/6602) Fixed an issue which caused a compiler error (duplicate class) when including `payments` *and* `identity` SDKs.
```

Reply:
`payments` isn't really an SDK that would be referenced in this way (it's referred to as `stripe-android`). I think it's best to leave it as is.